appium/tests/specs/mock/inbox/ReadTextEmail.spec.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  SplashScreen,
+  SetupKeyScreen,
+  MailFolderScreen,
+  EmailScreen,
+  SearchScreen
+} from '../../../screenobjects/all-screens';
+
+import { CommonData } from '../../../data';
+import { MockApi } from "../../../../api-mocks/mock";
+import { ekmKeySamples } from "../../../../api-mocks/apis/ekm/ekm-endpoints";
+
+describe('INBOX: ', () => {
+
+  it('user is able to view text email and recipients list', async () => {
+
+    const senderName = CommonData.recipientsListEmail.senderName;
+    const emailSubject = CommonData.recipientsListEmail.subject;
+    const emailText = CommonData.recipientsListEmail.message;
+    const recipientsButton = CommonData.recipientsListEmail.recipients;
+    const toLabel = CommonData.recipientsListEmail.to;
+    const ccLabel = CommonData.recipientsListEmail.cc;
+    const bccLabel = CommonData.recipientsListEmail.bcc;
+
+    const mockApi = new MockApi();
+    mockApi.fesConfig = {
+      clientConfiguration: {
+        flags: ["NO_PRV_CREATE", "NO_PRV_BACKUP", "NO_ATTESTER_SUBMIT", "PRV_AUTOIMPORT_OR_AUTOGEN", "FORBID_STORING_PASS_PHRASE"],
+        key_manager_url: CommonData.keyManagerURL.mockServer,
+      }
+    };
+    mockApi.ekmConfig = {
+      returnKeys: [ekmKeySamples.e2eValidKey.prv]
+    }
+
```

Comment:
This is good. I would like to take it a step further, and also have:

```ts
mockApi.gmailConfig = {
  accounts: {
    'e2e.enterprise.test@flowcrypt.com': {
      messages: ['message-export-171d138c8750863b.json']
    }
  }
}
```

So for each test, we would define which accounts and which messages will be available on the mock api. That will make it much easier to debug and understand tests later (learned this lesson on browser extension).

Reply:
Maybe better if we define `messages` like below and we call something like `messages: [GmailMessages.testMessageForCCAndBCCRecipientsSignature]` (So that we benefit from auto-complete + developer doesn't need to look through full messages json to check if he wants which message.


```
export const GmailMessages = {
  'testMessageForCCAndBCCRecipientsSignature': 'message-export-171d138c8750863b.json',
  ....
}
```

Let me know what you think

Reply:
I like this idea, it'll make listing of mock messages simpler and more clear 👍

Reply:
That's one possibility. Another possibility, because most of these messages actually come exported from their real Gmail API counterparts, would be to refer to them by the subject:

```ts
mockApi.gmailConfig = {
  accounts: {
    'e2e.enterprise.test@flowcrypt.com': {
      messages: ['sig from new Thunderbird v78 not recognized, html']
    }
  }
}
```

To find the file this refers to, you'd CTRL + F `sig from new Thunderbird v78 not recognized, html` in your workspace. But more importantly, if you want to develop / manually test on live Gmail, or cross-check on browser extension, you could now directly search it:

![image](https://user-images.githubusercontent.com/6306961/173846728-7b686395-6860-4e6d-bff7-5bfb36017aff.png)